ocs_ci/helpers/helpers.py

```diff
@@ -2790,3 +2791,42 @@ def fetch_used_size(cbp_name, exp_val=None):
             f"matching. Retrying"
         )
     return used_in_gb
+
+
+def check_pvs_present_for_ocs_expansion(sc="localblock"):
+    """
+    Check for pvs present for OCS cluster expansion
+
+    Args:
+        sc (str): Name of SC
+
+
+    Returns:
+        bool: True if PVS present, assert if PVS not present
+
+    """
+    for rack_no in range(0, 3):
```

Comment:
any chance there will be other than 3 racks? How about KNIP-1477?

Reply:
for now, I'm not considering that

Reply:
also, will this check still apply in 4.7 where we will not have the rack labels?